services/provider/server/server.go

```diff
@@ -734,7 +735,8 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 		}
 	}
 
-	clientProfile := util.CalculateMD5Hash(req.StorageClaimName)
+	clientProfilemd5Sum := md5.Sum([]byte(req.StorageClaimName))
+	clientProfile := hex.EncodeToString(clientProfilemd5Sum[:])
```

Comment:
client op uses a different hash function, from what we have at CalculateMD5Hash, hence using this to make sure the clientProfileName is same that we have on the client side